specificatie/kadaster-natuurlijk-personen.yaml

```diff
@@ -115,6 +115,8 @@ components:
           type: array
           items:
             $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/HalLink"
+        zakelijkGerechtigdenCollectie:
```

Comment:
Ik denk dat er hier of in de description van de openapi.yaml wat documentatie (omdat deze property ook is gespecificeerd in kadastraal-onroerende-zaken.yaml) bij moet dat vertelt wat het doel is van deze property